auth/pyproject.rest.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "gcloud-rest-auth"
+version = "3.8.0"
```

Comment:
I'll do a major release of all of 'em post-merge, since they'll cascade (eg. need to release auth, then bump the others to use it, then release them...). Only reason auth got bumped to a prerelease is that I needed to use it for testing the changes tot he deployment CI task.